python-sdk/deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+python3 setup.py register -r pypitest
+python3 setup.py sdist upload -r pypitest
```

Comment:
Python3 like Scala 12 may be problematic.

On Ubuntu you call python3 "python3" on other 'nix (like Mac BSD Unix) it replaces the default "python". How do we handle this so that it will run on any machine? Perhaps some kind of aliasing setup in the ENV before any of these are called? Can you propose something that will work on all important machines, Debian derived, Centos derived, Mac?